app/src/main/java/org/wikipedia/page/Section.kt

```diff
@@ -0,0 +1,44 @@
+package org.wikipedia.page
+
+import com.google.gson.annotations.SerializedName
+import org.wikipedia.json.GsonUtil
+import java.util.*
+
+data class Section(var id: Int = 0,
```

Comment:
Can we have the parameters in the same line and only break to newline when the char limit on the same line is reached?

Reply:
Not sure but I am following the way we convert in the past for the `data class`.  I am also open to update to the way you've suggested.
https://github.com/wikimedia/apps-android-wikipedia/blob/2923c466f991e96f1e0a5119c15d510d08a19e6a/app/src/main/java/org/wikipedia/page/references/PageReferences.kt
https://github.com/wikimedia/apps-android-wikipedia/blob/2923c466f991e96f1e0a5119c15d510d08a19e6a/app/src/main/java/org/wikipedia/page/references/PageReferences.kt